### ISSUE TITLE
Resolve confusion around preview build versioning

### ISSUE THREAD

**Issue by mthalman** [dotnet/source-build#3521]
In our announcements for Preview releases, we specify the build version in the tag and announcement text. However, this build version is the version of the Microsoft build which is not necessarily the same version that you would get as a result of a source-built SDK. This leads to confusion because the build versions do not align. See https://github.com/dotnet/source-build/discussions/3504#discussioncomment-6192224.

The reason for why the build versions don't align is due to Arcade behavior and described here: https://github.com/dotnet/source-build/issues/3248.

We should investigate ways to resolve this confusion and implement a solution.

**Comment by mthalman** [comment]
Using stable versioning for our preview releases would also resolve this issue. cc @mmitche

**Comment by mmitche** [comment]
Yes agreed that this would solve this problem (at least from an outward branding perspective)
